routing/index_graph.hpp

```diff
@@ -37,6 +39,11 @@ class IndexGraph final
   using Edge = SegmentEdge;
   using Weight = RouteWeight;
 
+  // Aliases for JointEdge retrieving. The number of elems in such structures is usually 3-4 items.
+  // So we use buffer_vector instead of std::vector to excludes to much operator new(...) invocations.
```

Comment:
... to exclude too many operator new(...) invocations.

Reply:
done